.gitignore

```diff
@@ -0,0 +1 @@
+node_modules/
```

Comment:
There are best practice out there, for example, https://github.com/Wiredcraft/api-boilerplate/blob/master/.gitignore, https://github.com/github/gitignore